.github/install_examples.sh

```diff
@@ -2,6 +2,22 @@
 # Script that generates the required target files for each example project
 JAVA_MAJOR_VERSION=$(java -version 2>&1 | head -1 | cut -d'"' -f2 | sed 's/^1\.//' | cut -d'.' -f1)
 
+# Java version needs to be lower than 11, since support was dropped
+# If the source version is 1.1 OR 1.2, we can only install JUnit3 tests AND we need to use 1.3 as the source version since Java 8 does not support earlier versions
```

Comment:
> AND we need to use 1.3 as the source version since Java 8 does not support earlier versions

Do you mean source 1.3?  (Line 8 has `-Dmaven.compiler.source="1.2"`)

Reply:
Yes I meant source 1.3, since the official docs say it only supports 1.3 upwards:

```
-source release

    Specifies the version of source code accepted. The following values for release are allowed:

    1.3

        The compiler does not support assertions, generics, or other language features introduced after Java SE 1.3.
    1.4

        The compiler accepts code containing assertions, which were introduced in Java SE 1.4.
    1.5

        The compiler accepts code containing generics and other language features introduced in Java SE 5.
    5

        Synonym for 1.5.
    1.6

        No language changes were introduced in Java SE 6. However, encoding errors in source files are now reported as errors instead of warnings as in earlier releases of Java Platform, Standard Edition.
    6

        Synonym for 1.6.
    1.7

        The compiler accepts code with features introduced in Java SE 7.
    7

        Synonym for 1.7.
    1.8

        This is the default value. The compiler accepts code with features introduced in Java SE 8.
    8

        Synonym for 1.8.

```

But, in reality, it works and compiles.

Probably a good idea to set it to `1.3` tho.